docs/reference/errors-and-warnings/NU5035.md

```diff
@@ -0,0 +1,21 @@
+---
+title: NuGet Error NU5035
+description: NU5035 Error code
+author: nkolev92
+ms.date: 11/15/2018
+ms.topic: reference
+ms.reviewer: karann
+f1_keywords: 
+  - NU5035
+---
+
+# NuGet Error NU5035
+<pre>The PackageLicenseUrl cannot be used cannot be used in conjunction with the PackageLicenseFile and PackageLicenseExpression.</pre>
+
+### Issue
+
+When using PackageLicenseFile, PackageLicenseExpression or directly the license element in the nuspec, you should not use the licenseUrl metadata. It will be auto-generated by the client tools to a down-level friendly url.
```

Comment:
> When using PackageLicenseFile, PackageLicenseExpression or directly the license element in the nuspec

The bit after the word 'or' needs to be improved/fixed.

Reply:
I've reworded it. 
It was needlessly verbose earlier.